.github/workflows/autoreview.yml

```diff
@@ -10,6 +10,15 @@ jobs:
       uses: reviewdog/action-misspell@v1
       with:
         pattern: "*.md"
+  checklinks:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Link Checker
+        uses: lycheeverse/lychee-action@v1.8.0
+        with:
+          args: --verbose './**/*.md' 
+          fail: false
```

Comment:
Any specific reason you don't fail the action if the link check fails? Would it still give an in-your-face warning? (Just to avoid a PR gets merged even though there are broken links.)

Reply:
Just in case the checklinks job completes before the other ones, I didn't want the other jobs (in autoreview) getting cancelled. I figured rather than failing the entire Job, a human reviewer could perhaps look at them instead.

There should still be a summary in actions page that looks like below. But I suppose this can be furthered developed into perhaps creating auto issues in the repo.

<img width="604" alt="image" src="https://github.com/AntennaPod/antennapod.github.io/assets/29858077/82c46a86-e9d9-4bcb-9d17-77ce25b82ae5">

Reply:
My experience is that a check that does not fail is basically useless because one doesn't always open the job output page before merging if there is already a green checkmark

Reply:
Ok. @shinwookim, would you mind changing it after all? If I'm not mistaken, only that linkcheck will then fail; it wouldn't prevent other tests in autoreview from running. And if needed, we can still merge PRs despite a failing test.